GETTING_STARTED_GUIDE.md

```diff
@@ -0,0 +1,153 @@
+## Getting started guide
+
+### How to run assistant on your own machine
+Below are the set of steps to run OpenSearch and OpenSearch dashboards with the OpenSearch assistant and the query generation functionality in the Observability Log Explorer page correctly on the cluster.
+**Note** that the `feature/langchain` is the branch used in this guide.
+
+1. Follow steps here to setup docker for OpenSearch: https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/
+   1. Note: When running docker pull, use this command instead: `docker pull public.ecr.aws/w1m7p7g2/opensearch-reinvent2023:latest`
+   2. Note: This docker image may get incrementally updated over time.
+
+
+2. Follow steps here to setup docker to OpenSearch Dashboards: https://opensearch.org/docs/latest/install-and-configure/install-dashboards/docker/
+   1. Note: When running docker pull, use this command instead for OSD: `docker pull public.ecr.aws/w1m7p7g2/opensearch-dashboards-reinvent2023:latest`
+   2. Note: This docker image may get incrementally updated over time.
+   3. If you want to enable the chat assistant feature, set `assistant.chat.enabled` to `true` in the `opensearch_dashboards.yml` file.
+3. After OpenSearch and OpenSearch Dashboards are running, we will setup ML Commons to connect to the LLM model
+4. Run ML commons on Data node
+   ```
+   PUT _cluster/settings
+   {
+     "persistent" : {
+     "plugins.ml_commons.only_run_on_ml_node":"false"
+     }
+   }
+     ```
+5. Add Trusted Endpoints ([reference doc](https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/index/))
+   ```
+   PUT _cluster/settings
+   {
+     "persistent" : {
+       "plugins.ml_commons.trusted_connector_endpoints_regex":
+       [ "^https://runtime\\.sagemaker\\..*[a-z0-9-]\\.amazonaws\\.com/.*$",
+         "^https://api\\.openai\\.com/.*$",
+         "^https://api\\.cohere\\.ai/.*$",
+         "^https://bedrock-runtime\\.us-east-1\\.amazonaws\\.com/.*$"
+       ]
+     }
```

Comment:
![image](https://github.com/opensearch-project/dashboards-assistant/assets/13493605/89c7074c-1cb7-45f4-8e67-3a479788c026)

Seems to break link checker. Perhaps we need to remove link checker.

Reply:
good point, we can exclude links with `^https` which is probably regex. i'll send a separate PR since it's not part of the merge

Reply:
https://github.com/opensearch-project/dashboards-assistant/pull/73